.github/workflows/eslint.yml

```diff
@@ -0,0 +1,35 @@
+name: eslint
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install eslint and formatter
+        run: npm i eslint @microsoft/eslint-formatter-sarif
+          
+      - name: Run eslint
+        run: npx eslint .
+             --format @microsoft/eslint-formatter-sarif 
+             --output-file eslint.sarif
+        continue-on-error: true
+        
+      - name: Upload SARIF to GitHub
+        uses: github/codeql-action/upload-sarif@v1
+        with:
+          sarif_file: eslint.sarif
+          
+      - name: Upload eslint results as artifact
```

Comment:
Last nit: You say "Upload SARIF" and "Upload eslint" here? It's the same thing right?

Reply:
I updated the message.